listenstore/listenstore/listenstore.py

```diff
@@ -45,6 +46,10 @@ def fetch_listens_from_storage():
         """ Override this method in PostgresListenStore class """
         raise NotImplementedError()
 
+    def get_listen_count_for_user(self, user_name, need_exact):
```

Comment:
This should have documentation to say what the `need_exact` field does.

Does it make sense to have the function in this class do the redis estimate value and call the more specific implementation only if needed?